config/betterquesting/DefaultQuests.json

```diff
@@ -77350,7 +77350,7 @@
           "lockedProgress:1": 0,
           "autoClaim:1": 0,
           "isSilent:1": 0,
-          "desc:8": "Cracking oil products like light and heavy fuel with the oil cracker is faster and more efficent than using a chemical reactor, producing 25 percent more output. Cracked products will result in additional by-products in the distillation tower. It is a smart idea to get this multiblock since you are going to need TONS of industrial TNT from Toluene for advanced tier rockets. \n\nThe 16 coils form 2 vertical rings of 8 on either side of the controller. The casings between the rings can be replaced with a hatch for steam/hydrogen input. Energy hatch and maintenance hatch can replace any of the casings. The input product hatch and output product hatch must replace one of the casings outside of the coils.\n\nHint: Only GT/Railcraft steam can be used, not ic2 steam. You can use a fluid heater to make it instead of a boiler.\n\n§3The circuit goes in the controller.\n\n§3PS: I listed all the cracker recipes on the spreadsheet for you to look at in a convenient format. Now you can easily compare which oil path to take!\n\n§2Bart change the multi so it has locked I/O now, go badger him to fix the I/O on the sides or tell him you want to be able to input the steam/hydrogen from anywhere other than the top if you don\u0027t like it."
+          "desc:8": "Cracking oil products like light and heavy fuel with the oil cracker is faster and more efficent than using a chemical reactor, producing 25 to 50 percent more output, depending on which cracking method it\u0027s doing (steam does 50%, hydro does 25% but consumes 20% less hydrogen). Cracked products will result in additional by-products in the distillation tower. It is a smart idea to get this multiblock since you are going to need TONS of industrial TNT from Toluene for advanced tier rockets. \n\nThe 16 coils form 2 vertical rings of 8 on either side of the controller. The casings between the rings can be replaced with a hatch for steam/hydrogen input. Energy hatch and maintenance hatch can replace any of the casings. The input product hatch and output product hatch must replace one of the casings outside of the coils.\n\nHint: Only GT/Railcraft steam can be used, not ic2 steam. You can use a fluid heater to make it instead of a boiler.\n\n§3The circuit goes in the controller.\n\n§3PS: I listed all the cracker recipes on the spreadsheet for you to look at in a convenient format. Now you can easily compare which oil path to take!\n\n§2Bart change the multi so it has locked I/O now, go badger him to fix the I/O on the sides or tell him you want to be able to input the steam/hydrogen from anywhere other than the top if you don\u0027t like it."
```

Comment:
Bart change the  should be "Bart changed the"

That line may need rework as Bart is retired so ... ?

Reply:
I removed that portion of the description, since I tested the Oil Cracker when changing it and I was inputting Steam and Hydrogen from the back just fine.